### PR TITLE
Tweak rules

### PR DESCRIPTION
**Pull request by saas786**
https://app.clickup.com/t/3gfbyhx

### CODE REVIEW

File: CXL/ruleset.xml [hunk 1]
```diff
@@ -147,6 +147,8 @@
         <exclude name="SlevomatCodingStandard.Commenting.DisallowCommentAfterCode"/>
         <!-- Multi or single line are both fine. Feel free to remove this exclusion if you prefer to enforce single line where they're possible -->
         <exclude name="SlevomatCodingStandard.Commenting.DisallowOneLinePropertyDocComment.OneLinePropertyComment"/>
+        <!-- Class method we allow, One Line Doc Comment where there's only 1 annotation present -->
```
Comment (pawelkmpt) [inline CXL/ruleset.xml]: This comment is unclear for me. Did you mean:

```suggestion
        <!-- We allow multiline doc comment for class when there's only 1 annotation present -->
```

?

**Review by pawelkmpt — COMMENTED** [review]
Branch name should be `syed/chore/slevomat`

File: CXL/ruleset.xml (hunk 1 above)
Comment (pawelkmpt) [reply CXL/ruleset.xml]: Actually it should've been:

```diff
- We allow multiline doc comment for class when there's only 1 annotation present
+ We allow multiline doc comment for class or method when there's only 1 annotation present
```

**Review by pawelkmpt — CHANGES_REQUESTED** [review]
What did you change in recent push? I don't see ruleset comment updated.

Please also update commit message in 782d7a38e0bdb72b6a795e94a53f07c1967c6128
and add "closes #4"

**Comment by saas786** [comment]
@pawelkmpt 

> Please also update commit message in https://github.com/conversionxl/phpcs-ruleset/commit/782d7a38e0bdb72b6a795e94a53f07c1967c6128

Update commit message to 

from
**(chore: Add `@inheritDoc` to `annotationsGroups` rule)**
to
**`?`**

**Comment by pawelkmpt** [comment]
> @pawelkmpt
> 
> > Please also update commit message in [782d7a3](https://github.com/conversionxl/phpcs-ruleset/commit/782d7a38e0bdb72b6a795e94a53f07c1967c6128)
> 
> Update commit message to
> 
> from **(chore: Add `@inheritDoc` to `annotationsGroups` rule)** to **`?`**

I think I pointed the wrong commit. I meant the one with `RequireOneLineDocComment` to match commit message with comment in xml. I see you updated it now 👍

**Review by pawelkmpt — APPROVED** [review]

**Pull request merged**